src/auth/auth.controller.ts

```diff
@@ -54,4 +61,78 @@ export class AuthController {
 
     res.sendStatus(HttpStatus.OK);
   }
+
+  @ApiExcludeEndpoint()
+  @Get('login')
+  async jwtLogin(
+    @Query(
+      new ValidationPipe({
+        errorHttpStatusCode: HttpStatus.UNPROCESSABLE_ENTITY,
+        transform: true,
+      }),
+    )
+    { token }: LoginQueryDto,
+    @Res() res: Response,
+  ): Promise<void> {
+    if (this.config.get<boolean>('DISABLE_LOGIN')) {
+      throw new UnauthorizedException();
+    }
+
+    const email = this.verifyEmailFromJwt(token);
+
+    const user = await this.usersService.findByEmail(email);
+    if (user) {
+      await this.usersService.updateLastLoginAt(user);
+    } else {
+      throw new UnauthorizedException({ error: 'user_invalid' });
```

Comment:
We need to use redirect to throw error because this will be user facing. 
Im thinking redirect to https://testnet.ironfish.network/login?toast=encrypted error message

Reply:
Good call, updated